### PR TITLE
Multinode support

### PR DESCRIPTION
**Pull request by edcdavid**
Adding multinode support (useful for running anti-affinity tests)

### CODE REVIEW

**Review by jun-rh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -120,11 +122,12 @@ To verify `partner` and `test` pods are running:
...
+You should see something like this (note that the test pods are running on different nodes due to a anti affinity rule):
 ```shell-script
-NAME      READY   STATUS    RESTARTS   AGE   IP           NODE       NOMINATED NODE   READINESS GATES
-partner   1/1     Running   0          22m   172.17.0.3   minikube   <none>           <none>
-test      1/1     Running   0          22m   172.17.0.4   minikube   <none>           <none>
+NAME                       READY   STATUS    RESTARTS   AGE    IP           NODE           NOMINATED NODE   READINESS GATES
+partner-68cf756959-tp2c5   1/1     Running   0          110s   10.244.1.7   minikube-m02   <none>           <none>
+test-7799cc9677-6d8qz      1/1     Running   0          110s   10.244.1.8   minikube-m02   <none>           <none>
+test-7799cc9677-rv8nv      1/1     Running   0          110s   10.244.0.5   minikube       <none>           <none>
 ```
```
Comment (wying3) [inline README.md:131]: can this to be seen from workflow output logs?

Comment (edcdavid) [reply README.md:131]: I added the log in the workflow output

File: .github/actions/create-local-test-infra-resources/action.yaml [hunk 2]
```diff
@@ -44,6 +44,10 @@ runs:
       run: oc wait pod --for=condition=ready -l "app=${{ inputs.tpp_pod_label_app }}" --timeout=${{ inputs.oc_pod_timeout }}
       shell: bash
 
+    - name: Display all pods in namespace
+      run: oc get pods -o wide
```
Comment (wying3) [inline .github/actions/create-local-test-infra-resources/action.yaml:48]: it need -A or  -n namespace, otherwise no output

Comment (edcdavid) [reply .github/actions/create-local-test-infra-resources/action.yaml:48]: The namespace is set in the lines above

**Review by wying3 — APPROVED** [review]

**Review by jun-rh — APPROVED** [review]

**Review by HamadiSE — APPROVED** [review]

**Pull request merged**